.env.test

```diff
@@ -1,6 +1,7 @@
 DEBUG=true
 SECRET_KEY='tests'
 DATABASE_URL='postgres://localhost/opencraft'
+INSTANCE_MYSQL_URL='mysql://root@localhost'
```

Comment:
It's not currently needed for the tests. But I will add it when I write the logic for creating mongo users